src/borg/testsuite/archiver.py

```diff
@@ -2793,6 +2793,7 @@ def test_config(self):
             assert output == cfg_value + '\n'
             self.cmd('config', '--delete', self.repository_location, cfg_key)
             self.cmd('config', self.repository_location, cfg_key, exit_code=1)
+            self.cmd('config', '--list', '--delete', self.repository_location, exit_code=2)
```

Comment:
doesn't look like there is a test whether --list works as expected.

Reply:
I'll have a look at it.
The problem I have is that some values change at each new repo (repo id for example). I'll try to find a way to check that it works as expected.